server/implementation/src/main/java/io/smallrye/graphql/execution/datafetcher/AbstractDataFetcher.java

```diff
@@ -48,10 +43,6 @@ public AbstractDataFetcher(Operation operation, Config config) {
         this.argumentHelper = new ArgumentHelper(operation.getArguments());
         this.partialResultHelper = new PartialResultHelper();
         this.batchLoaderHelper = new BatchLoaderHelper();
-        if (config != null && config.getUnwrapExceptions().isPresent()) {
```

Comment:
I would do this with an `ifPresent` block, but that's just a matter of taste, I guess.

Reply:
This is being deleted :)